horace_core/sqw/coord_transform/@ortho_proj/ortho_proj.m

```diff
@@ -520,6 +504,79 @@
             %          Expressed in degree
             [u,v,w,type,nonortho] = uv_from_rlu_mat_(obj,u_rot_mat,ulen);
         end
+    end
+    methods(Static)
+        function lst = data_sqw_dnd_export_list()
+            % TODO: temporary method, which define the values to be
+            % extracted from projection to convert to old style data_sqw_dnd
+            % class. New data_sqw_dnd class will contain the whole projection
+            lst = {'u_to_rlu','nonorthogonal','alatt','angdeg','uoffset','label'};
+        end
+    end
+    %=====================================================================
+    % SERIALIZABLE INTERFACE
+    %----------------------------------------------------------------------
+    methods
+        % check interdependent projection arguments
+        function wout = check_combo_arg (w)
+            % Check validity of interdependent fields
+            %
+            %   >> [ok, mess,obj] = check_combo_arg(w)
+            %
+            % Throws HORACE:ortho_proj:invalid_argument with the message
+            % suggesting the reason for failure if the inputs are incorret
+            % w.r.t. each other.
+            % 
+            wout = check_combo_arg_(w);
+        end
+        %------------------------------------------------------------------
+        function ver  = classVersion(~)
+            ver = 1;
+        end
+        function  flds = saveableFields(obj)
+            flds = saveableFields@aProjection(obj);
+            flds = [flds(:);obj.fields_to_save_(:)];
+        end
+    end
+    properties(Constant, Access=private)
+        fields_to_save_ = {'u','v','w','nonorthogonal','type'}
+    end
+    methods(Static)
+        function obj = loadobj(S)
+            % boilerplate loadobj method, calling generic method of
+            % savable class
+            obj = ortho_proj();
+            obj = loadobj@serializable(S,obj);
+        end
+        %
+        function proj = get_from_old_data(data_struct,header_av)
+            % construct ortho_proj from old style data structure
+            % normally stored in binary Horace files versions 3 and lower.
+            %
+            proj = ortho_proj();
+            if ~exist('header_av','var')
+                header_av = [];
+            end
+            proj = proj.from_old_struct(data_struct,header_av);
+        end
+    end
+    methods(Access=protected)
+        function obj = from_old_struct(obj,inputs,varargin)
+            % Restore object from the old structure, which describes the
+            % previous version of the object.
+            %
+            % The method is called by loadobj in the case if the input
+            % structure does not contain a version or the version, stored
+            % in the structure does not correspond to the current version
+            % of the class.
 
+            if nargin <= 2
+                header_av = [];
+            else
+                header_av = varargin{1};
+            end
```

Comment:
```suggestion
            if isempty(varargin)
                header_av = [];
            else
                header_av = varargin{1};
            end
```
Better yet, pass `header_av` explicitly and 
```
if ~exist('header_av', 'var')
    header_av = [];
end
```

Reply:
to be fair, petty changes, but done as playing with ortho_proj anyway